easy/4.js

```diff
@@ -1,5 +1,13 @@
 function matchHouses(step) {
-	
+   
+    if(step==0){
+        return 0;
+    }
+
+    let returnValue = (step*5) + 1;
+    //console.log(returnValue);
+
+    return returnValue;

```

Comment:
You don't have to declare this variable if it exists only for store value and return it